docs/source/blog/6-costly-data-labeling-mistakes-and-how-to-avoid-them.md

```diff
@@ -0,0 +1,127 @@
+---
+
+In data-centric artificial intelligence (data-centric AI), the model is only as good as the data that trains it. And the secret to high-quality training data? High-quality data labeling. Before a model can make any sense of a raw dataset (videos, images, text files, etc.), human annotators must provide the layer of meaning that contextualizes the data.   
+
+And humans … well, we don’t always get it right. [A recent MIT study](https://arxiv.org/abs/2005.11295) found that even popular public training datasets like ImageNet have a significant rate of data labeling errors, which it documented on the website [Label Errors](https://labelerrors.com/). When and why do data labeling mistakes like this happen? 
+
+It’s not always as simple as blaming the challenges on “human error”—flaws in the data annotation pipeline are often the root causes of mislabeled data. Thankfully, you have control over this factor when you’re managing a machine learning project. 
+
+Below are six of the most common and costly data labeling mistakes we see in machine learning projects and the fixes that can help you maintain consistent, accurate training data throughout the project pipeline.
+
+## 1. Missing Labels
+
+Missing labels can happen with any type of raw data, but they’re especially common during image classification. If the annotator neglects to put a bounding box around every necessary object in an image, they’ve created a missing label error. And while the mistake might sound simple, the implications can be very serious: Consider the self-driving car that can’t recognize every pedestrian.
+
+Let’s take a look at an image pulled from MIT’s [Label Errors](https://labelerrors.com/) website. The image was labeled by the ImageNet algorithm, so we don’t actually know which annotation error(s) led to the algorithm’s incorrect prediction, but it can give us a glimpse of why labels get missed.
+<br/>  
+
+<img src="/images/6-mistakes/great-dane-label-error.png" alt="Image of label studio timeline" style="max-width: 20rem;" />
+
+*Image Source: [Label Errors](https://labelerrors.com/)*
+
+At first glance, it’s hard to tell how many dogs are in the picture. The dalmatian in the foreground is clear, but the Great Dane’s face is obscured. What’s more, there are two *additional* dogs in the photo; they’re just harder to spot. If you gave this image to an annotator and told them to place a bounding box around every dog, it’s reasonable to assume your annotator might miss one or two dogs if they were fatigued or moving quickly. 
+
+### The Fix: Use the Consensus Method
+
+The key to catching and mitigating this kind of error is a strong, collaborative data validation process. The consensus method works great: Ask two or more different annotators to label the same sample of images, find the classes where annotators disagree, and revise your instructions until the annotators are able to find consistency. A thorough peer review process can also catch many quality errors. 
```

Comment:
Replace line #33 with the following: 

A strong, collaborative data validation process is the key to catching and mitigating this kind of error. The consensus method works great: Ask two or more different annotators to label the same sample of images, find the classes where annotators disagree, and revise your instructions until the annotators can find consistency. A thorough peer review process can also catch many quality errors.